### PR TITLE
add area to buildings

### PR DESCRIPTION
**Pull request by spiderbiggen**
This adds the area of a building as the last argument in the building

### CODE REVIEW

**Comment by nskartoredjo** [comment]
we approve -CTD

**Comment by lisv** [comment]
Should be very useful! Approved.
-CodeFox

**Comment by klikooo** [comment]
We approve - WhySoSerious

File: contextvh/src/test/java/contextvh/translators/ContextJ2BuildingsTest.java [hunk 1]
```diff
@@ -44,7 +44,7 @@
     @Test
     public void J2BuildingTest() throws TranslationException {
         GeometryFactory gf = new GeometryFactory();
-        MultiPolygon mp = gf.createMultiPolygon(new Polygon[0]);
+        MultiPolygon mp = PowerMockito.spy(gf.createMultiPolygon(new Polygon[0]));
         Collection<Category> categories = Arrays.asList(Category.EDUCATION, Category.BRIDGE);
 
         Building b = PowerMockito.mock(Building.class);
```
Comment (MaxGroenenboom) [inline contextvh/src/test/java/contextvh/translators/ContextJ2BuildingsTest.java]: Maybe refactor this name to a more understandable one?

**Comment by spiderbiggen** [comment]
approved by 3 groups , will merge

**Pull request merged**